src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -5,10 +5,11 @@
 Date-Time Cycling
 =================
 
-.. warning::
+.. admonition:: Aims
```

Comment:
Happy to do this on the fly for now, or to create a new admonition in cylc-sphinx-ext if you prefer, complete with re-styling `<li>`.

Reply:
We can leave as an admonition for now, easy to change later.